docs/microfrontend-composer/previous-tools/cms/colorare_il_cms.md

```diff
@@ -3,6 +3,11 @@ id: colorare_il_cms
 title:  Color the CMS
 sidebar_label: Color the CMS
 ---
+
+:::caution Deprecated
```

Comment:
This is fantastic! Should we provide an EOL timeline/version where the component will be removed?